product_seasonality/models/seasonal_config_line.py

```diff
@@ -54,7 +58,11 @@ def is_sale_ok(self, date):
 
     def find_for_product(self, prod, config=None):
         domain = [
+            "|",
             ("product_id", "=", prod.id),
+            "&",
+            ("product_id", "=", False),
+            ("product_template_id", "=", prod.product_tmpl_id.id),
```

Comment:
add a test pls

Reply:
Test already done by this little change:
https://github.com/OCA/product-attribute/pull/871/files#diff-ca80e35cf3f4f78c374cbae4e772aea98dad0bda75d76628eb3166dbe9f5503fR52

With it we have a line with product variant, and one without.